### DISCUSSION TITLE
Auto dark mode on chrome and microsoft is different

### DISCUSSION THREAD

**Discussion by KillingKarma** [AutoDarkMode/Windows-Auto-Night-Mode#776] [App recomendations]
I want to ask if anyone has the same situation as me?. When I download Auto Dark Mode to my computer. After installation was complete, I downloaded an extension for Chrome and Edge so that the web browser could detect the system mode. I installed the system theme automatic detection mode on Chrome and it worked. works properly with settings on Auto Dark Mode. As for Edge, I also installed the same as Chrome, but it doesn't work properly. When I select the extension's automatic mode on Edge, it turns on light while the Auto Dark Mode software turns on dark mode. While Chrome works properly with Auto Dark Mode software?
With one more question: How can I set from dark mode enabled from 6:00 PM to 6:00 AM?. I see that in the "dark transition time options" section, there are 2 adjustment bars for Bright and Dark. So when I set Dark to turn on at 6:00 PM, it will turn on at that time. As for the Light bar, if I set it to 6:00 AM, will it automatically switch to light mode at 6:00 AM or will it continue to turn on dark mode at 6:00 AM?. Can anyone guide me on how to set dark mode to turn on from 6:00 PM to 6:00 AM so it will automatically turn on light mode?

**Comment by Armin2208** [comment]
Just try out the times, it should work without problems
